admin-dev/themes/default/template/controllers/dashboard/helpers/view/view.tpl

```diff
@@ -87,34 +87,8 @@
 		<div class="col-md-4 col-lg-3" id="hookDashboardZoneOne">
 			{$hookDashboardZoneOne}
 		</div>
-		<div class="col-md-8 col-lg-7" id="hookDashboardZoneTwo">
+		<div class="col-md-8 col-lg-9" id="hookDashboardZoneTwo">
```

Comment:
Waiting this, it could be a conflict: https://github.com/PrestaShop/PrestaShop/pull/26239